Samples/excel-shared-runtime-global-state/README.md

```diff
@@ -0,0 +1,117 @@
+
+## Applies to
+
+-  Excel on Windows, Mac, and in a browser.
+
+## Prerequisites
+
+To try the preview features used by this sample, you will need to [join Office Insider](https://insider.office.com/join).
+
+Before running this sample, make sure you have installed a recent version of [npm](https://www.npmjs.com/get-npm) and [Node.js](https://nodejs.org/en/) on your computer. To check if you have already installed these tools, run the commands `node -v` and `npm -v` in your terminal.
+
+## Solution
+
+Solution | Author(s)
+---------|----------
+Office Add-in share global data with a shared runtime | Microsoft
+
+## Version history
+
+Version  | Date | Comments
+---------| -----| --------
+1.0 | 3-15-2020 | Initial release
+
+## Disclaimer
+
+**THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.**
+
+----------
+
+## Scenario: Sharing key/value pairs
+
+This sample enables a user to store and retrieve key/value pairs by using the task pane or custom functions. The user can select which type of storage is used. They can choose to store key/value pairs in local storage, or choose to use a global variable.
+
+## Build and run the solution
+
+In the command prompt, run the command `npm run start`. This will start the node server, and automatically open Excel on the desktop.
+If you are running Excel on the web or Mac, see the following articles for instructions on how to sideload:
+
+- [Sideload Office Add-ins in Office on the web for testing](https://docs.microsoft.com/office/dev/add-ins/testing/sideload-office-add-ins-for-testing) 
+- [Sideload Office Add-ins on iPad and Mac for testing](https://docs.microsoft.com/office/dev/add-ins/testing/sideload-an-office-add-in-on-ipad-and-mac)
+
+Once the add-in is loaded use the following steps to try out the functionality.
+
+1. On the `Home` tab, choose `Show TaskPane`.
+2. In the task pane, enter a key/value pair and choose `Store key/value pair`.
+![Screen shot of both key and value input fields, and both store and get buttons.](task-pane-buttons.png)
+3. In any spreadsheet cell, enter the formula `=CONTOSO.GETVALUEFORKEYCF("1")`. Pass the value of the key you created from the task pane.
+4. In any spreadsheet cell, enter the formula `=CONTOSO.SETVALUEFORKEYCF("2","oranges")`. The formula should return the text `Stored key/value pair`.
+5. In the task pane, enter the key from the previous formula `2` and choose `Get value for key`. The task pane should display the value `oranges`.
+
+The task pane and custom function are sharing data via a global variable in the shared runtime. You can switch the method of storage by choosing either the `Global variable` or `Local storage` radio buttons on the task pane.
+
+## Key parts of this sample
+
+The manifest.xml is configured to use the shared runtime by using the `Runtimes` element as follows:
```

Comment:
```suggestion
Configure the manifest.xml  to use the shared runtime by using the `Runtimes` element as follows:
```

Reply:
It's already configured. We give that to them as part of the sample.